.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": [
+    "react-app",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings",
+    "plugin:import/typescript"
+  ],
+  "rules": {
+    "@typescript-eslint/brace-style": ["error", "1tbs", { "allowSingleLine": true }],
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/func-call-spacing": ["error"],
+    "@typescript-eslint/member-ordering": ["warn"],
+    "@typescript-eslint/no-require-imports": ["error"],
+    "react/no-unused-prop-types": ["warn", { "skipShapeProps": true }],
+    "array-bracket-spacing": ["warn", "never"],
+    "import/order": [
+      "error",
+      {
+        "groups": [
+          "builtin",
+          "external",
+          ["internal", "parent", "sibling", "index"]
+        ],
+        "newlines-between": "always"
+      }
+    ],
+    "max-len": ["warn", { "code": 120 }],
+    "no-console": "warn",
+    "no-plusplus": "error",
+    "no-undef": "warn",
+    "object-curly-spacing": ["warn", "always"]
+  }
```

Comment:
Are these configs copied from a project? Was it a NextJs project? Do they include react-hooks linters? I'd also consider adding the a11y linter: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y

Reply:
I took these from `youth-membership` and checked against `palvelutarjotin-ui`. Seems there was not much difference between react and nextjs eslint files.

And to answer the above guestion I used `npx create-next-app`

Reply:
Added `"react/react-in-jsx-scope": "off",` and `    "globals": {
      "React": "writable"
    }`
These should disable the warning about import react to each component (which isn't required in NextJs)